content/guides/models/registry/configure_registry.md

```diff
@@ -7,107 +7,121 @@ title: Configure registry access
+| Create a new automation for a collection                       | Create           |        |   X    |   X   |
+| Turn on Slack notifications for new version being added        | Create           |        |   X    |   X   |
+| Create a new collection                                        | Create           |        |   X    |   X   |
+| Create a new custom registry                                   | Create           |        |   X    |   X   |
+| Edit collection card (description)                             | Update           |        |   X    |   X   |
+| Edit linked artifact description                               | Update           |        |   X    |   X   |
+| Add or delete a collection’s tag                               | Update           |        |   X    |   X   |
+| Add or delete an alias from a linked artifact                  | Update           |        |   X    |   X   |
+| Link a new artifact                                            | Update           |        |   X    |   X   |
+| Edit allowed types list for a registry                         | Update           |        |   X    |   X   |
+| Edit custom registry name                                      | Update           |        |   X    |   X   |
+| Delete a collection                                            | Delete           |        |   X    |   X   |
+| Delete an automation                                           | Delete           |        |   X    |   X   |
+| Unlink an artifact from a registry                             | Delete           |        |   X    |   X   |
+| Edit accepted artifact types for a registry                    | Admin            |        |        |   X   |
+| Change registry visibility (Organization or Restricted)        | Admin            |        |        |   X   |
+| Add users to a registry                                        | Admin            |        |        |   X   |
+| Assign or change a user's role in a registry                   | Admin            |        |        |   X   |
 
-{{< img src="/images/registry/configure_role_registry.gif" alt="" >}}
-
-## Remove a user from a registry
-1. Navigate to the **Registry** App in the W&B App UI.
-2. Select a core or custom registry.
-3. Click on the gear icon on the upper right hand corner.
-4. Scroll to the **Registry members and roles** section and type in the username of the member you want to remove.
-5. Click the **Delete** button.
 
+### Inherited permissions
 
+A user's permission in a registry depends on the highest level of privilege assigned to that user, whether through a team or individual. 
 
-## Registry visibility types
+For example, suppose a registry admin adds a user called Nico to Registry A and assigns them a `Viewer` registry role. The registry admin then adds a team called Foundation Model Team to Registry A and assigns Foundation Model Team a `Member` registry role. Nico is a member of Foundation Model Team.
```

Comment:
We are in a UI so I suspect that the roles should be bold rather than code font. I thin, this example would be better as a numbered list.